odl/ufunc_ops/ufunc_ops.py

```diff
@@ -0,0 +1,196 @@
+# Copyright 2014-2016 The ODL development group
+#
+# This file is part of ODL.
+#
+# ODL is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# ODL is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with ODL.  If not, see <http://www.gnu.org/licenses/>.
+
+"""Ufunc operators for ODL vectors."""
+
+# Imports for common Python 2/3 codebase
+from __future__ import print_function, division, absolute_import
+from future import standard_library
+standard_library.install_aliases()
+
+import numpy as np
+from odl.set.space import LinearSpace
+from odl.space import ProductSpace, fn
+from odl.operator import Operator, MultiplyOperator
+from odl.util.utility import is_int_dtype
+from odl.util.ufuncs import UFUNCS
+
+__all__ = ()
+
+
+def _is_integer_only_ufunc(name):
+    return 'shift' in name or 'bitwise' in name or name == 'invert'
+
+LINEAR_UFUNCS = ['conj', 'negative', 'rad2deg', 'deg2rad', 'add', 'subtract']
+
+RAW_INIT_DOCSTRING = """
+
+"""
+
+
+RAW_EXAMPLES_DOCSTRING = """
+Examples
+--------
+>>> import odl
+>>> space = odl.{space}
+>>> op = {name}(space)
+>>> print(op({arg}))
+{result}
+"""
+
+
+class UfuncOperator(Operator):
```

Comment:
This one contains nothing different from an ordinary `Operator`, right? But on the other hand maybe it would be interesting at some point to know that it is a ``UfuncOperator`? I think that you @adler-j and @kohr-h are more suited to sort that out.